cogboard-app/src/main/kotlin/com/cognifide/cogboard/security/LoginHandler.kt

```diff
@@ -78,6 +78,6 @@ class LoginHandler : RoutingHandlerFactory {
     }
 
     companion object {
-        private const val SESSION_DURATION_IN_SECONDS = 30 * 60
+        private const val SESSION_DURATION_IN_SECONDS = 2 * 60 * 60
```

Comment:
Is it 2 minutes? or 2 hours ? please rename field to for example TWO_HOURS

Reply:
It's a slippery slope, because when someone changes the constant's value to something different than two hours, its name stops being true.

As a general good practice it could be better to do it like this:
`        private const val SESSION_DURATION_IN_SECONDS = 2 * 60 * 60 // two hours`
which still gets the point across without tying value to name.

Reply:
2 hours as stated by AC in story. I will rename it.

Reply:
Are we renaming it then or using comment to describe variable?

Reply:
@Southeaven pick your poison, either option is valid in its own right

Reply:
Do not rename it. I agree with @tMaxx to not couple the names with values.

Reply:
Added comment